Method: <org.apache.commons.math.MathException: java.lang.Throwable getCause()>, Callee: {cachedResources=null, pattern=Maximal number of iterations ({0}) exceeded, arguments=[10000], rootCause=null, class$java$lang$Throwable=class java.lang.Throwable, maxIterations=10000,}, Params: [], Related test: test25(org.apache.commons.math.distribution.Math103AB_1_Test)